lib/stack_master/stack_definition.rb

```diff
@@ -19,7 +21,12 @@ def template_file_path
     end
 
     def parameter_files
-      [ default_parameter_file_path, region_parameter_file_path ] + additional_parameter_lookup_file_paths
```

Comment:
FYI, I'm not entirely clear on the purpose of "additional_parameter_lookup_file_paths" vs as I did it below, explicitly resolving explicitly to the region_alias as I did below. I've preserved both, but I'm interested in the rationale behind additional_parameter_lookup_file_paths.